geometry-data-service.md

```diff
@@ -0,0 +1,94 @@
+# Geometry Data Service
+
+The Geometry Data Service (GDS) is a lightweight API that has been developed in Loc-I to facilitate 
+* minting URIs for geometries to enable Linked Data functionality
+* content-negotiation for formats and representations of the geometry
+* search by point for relevant geometries across the geometry store
+* parameterise the profile of the geometry representation for a Loc-I Feature (geometry of a feature with a resolution in metres, bounding box, centroid, simplified vs. complex geom, projection/CRS)
+
+![Geometry Data Service architecture](images/geometry-data-service.png "Geometry Data Service architecture")
+
+The current Geometry Data Service uses the [pyLDAPI](https://github.com/RDFLib/pyLDAPI) to implement the Linked Data APIs.
+
+
+
+# Scope
+
+##  Mint URIs for geometries and provide an API for it
+
+* Only store (geo-referenced) geometries
+* Scoped to Earth initially
+
+Implies 
+* Ability to embed a URI Reference in a Loc-I feature instance 
+* Selecting a predicate to associate a feature with the geometry URI reference (GeoSPARQL or something else?)
+* Returning the geometry representation from somewhere (from a remote service or local geodatabase cache)
+* Content negotiation by profiles
+
+
+## Out of scope
+
+Performing extensive utility calculations on geometries (i.e. [ArcGIS GeometryService](https://www.arcgis.com/home/item.html?id=2e18b487043641538f02028cc2495c0e)).
+
+
+# Content negotiation interfaces
+
+The Geometry Data Service implements and extends conneg arrangement proposed by (Regalia et al. 2017). 
+Conneg interfaces the GDS implements are shown below:
+
+| MIME Type | Description | Returns |
+|--------------|--------------|---------|
+|text/html |Web interface |`<!DOCTYPE html><html lang="en">...`|
+|text/plain |Well-Known Text |`POLYGON((113.1016 -38.062 ...))`|
+|application/json | GeoJSON |`{"type":"Polygon","coordinates":...}`|
```

Comment:
application/geo+json ?

Reply:
We currently haven't implemented application/geo+json, but we could add an issue to do so. Perhaps we can document that intent in this doc?

Reply:
See https://github.com/CSIRO-enviro-informatics/geometry-data-service/issues/15